quickstart.sh

```diff
@@ -67,9 +69,11 @@ fi
 
 if ! $local_deploy; then
   echo -e "${MAJOR}Updating configuration files for online deploy${RESET}"
-  sed -i.bu 's/localhost/chorus-opensearch-edition.dev.o19s.com/g'  ./chorus_ui/src/Logs.js
-  sed -i.bu 's/localhost/chorus-opensearch-edition.dev.o19s.com/g'  ./chorus_ui/src/App.js
-  sed -i.bu 's/localhost/chorus-opensearch-edition.dev.o19s.com/g'  ./opensearch/wait-for-os.sh
+  sed -i.bu "s/localhost/${hostname_or_public_ip}/g"  ./chorus_ui/src/Logs.js
+  sed -i.bu "s/localhost/${hostname_or_public_ip}/g"  ./chorus_ui/src/App.js
+  sed -i.bu "s/127.0.0.1/${hostname_or_public_ip}/g"  ./chorus_ui/src/App.js
```

Comment:
do we still need the 127.0.01?   Or was that the werid case in cors land?   Feels like we should just use localhost everywhere IF we can..